understanding/focus-appearance.html

```diff
@@ -445,498 +372,379 @@ <h4>Inline links</h4>
                   </p>
                   
                </section>
                <section id="keyboard-focus">
                   
-                  <h3>Component keyboard focus</h3>
+                  <h3>コンポーネントのキーボードフォーカス</h3>
                   
-                  <p>The preamble to this Success Criterion is "When a user interface component has keyboard
-                     focus..." The <em>keyboard focus</em> is the point of interaction for someone using a keyboard. For environments with a
-                     keyboard-operable interface, the keyboard focus can be moved around the interface
-                     in order to interact with different components. Whichever component is being interacted
-                     with has focus.
+                  <p>この達成基準の前文は、「When a user interface component have keyboard focus…」である。<em>キーボードフォーカス</em>は、キーボードを使用する人にとってインタラクションのポイントである。キーボードで操作可能なインタフェースのある環境では、様々なコンポーネントを操作するために、キーボードフォーカスをインタフェース内で移動させることができる。操作しているどのコンポーネントにもフォーカスがある。
                   </p>
                   
                   
-                  <p>WCAG defines <em>user interface component</em> as "a part of the content that is perceived by users as a single control for a distinct
-                     function." Because different users may perceive controls differently, there is a potential
-                     for some variation when interpreting what constitutes both a <em>single control</em> and a <em>distinct function</em>. This is particularly the case when something visually presents in a way that may
-                     differ from how it is programmatically created under the covers. Where there is not
-                     a native HTML component upon which to base designs, there can be great variations
-                     in how the components and their focus indicators are portrayed. Further, some components
-                     have sub-components that can take focus, such as the menu items on a menu.
+                  <p>WCAG は、<em>ユーザインタフェース コンポーネント</em>を「コンテンツの一部分で、特定の機能を実現するための単一のコントロールとして利用者が知覚するもの。」と定義している。利用者によってコントロールの捉え方が異なる場合があるため、何が<em>単一のコントロール</em>と<em>個別の機能</em>の両方を構成するかを解釈するときに、多少の差異が生じる可能性がある。これは特に、視覚的に表示されるものが、隠れてプログラムによって作成される方法とは異なる場合がある。設計のベースとなるネイティブ HTML コンポーネントがない場合、コンポーネントとそのフォーカスインジケータがどのように表示されるかは大きな差異がある可能性がある。さらに、メニューのメニュー項目など、いくつかのコンポーネントは、フォーカスを取得できるサブコンポーネントを持っている。
                   </p>
                   
                   
-                  <p>Nonetheless, consistent results from different testers were obtained for this Success
-                     Criterion by using the focus indicator itself as the gauge of what constitutes the
-                     component being interacted with. For complex components, the three typical focus indicators
-                     are as follows:
+                  <p>それにもかかわらず、この達成基準では、フォーカスインジケータそのものを、操作されるコンポーネントを構成するものの基準として使用することで、様々なテスターから一貫した結果が得られた。複雑なコンポーネントの場合、典型的なフォーカス指標は次の三つである:
```

Comment:
「フォーカス指標」は「フォーカスインジケータ」でよいと思います。

```suggestion
                  <p>それにもかかわらず、この達成基準では、フォーカスインジケータそのものを、操作されるコンポーネントを構成するものの基準として使用することで、様々なテスターから一貫した結果が得られた。複雑なコンポーネントの場合、典型的なフォーカスインジケータは次の三つである:
```